### DISCUSSION TITLE
Can I not appear on yhype.me ?

### DISCUSSION THREAD

**Discussion by arthaud-proust** [yhype/yhype-feedback#36] [Q&A] [answered]
Hi, 

I found my face on your site, because I have a github account i guess. I would like to be removed, is it possible ? 

Thanks,
Arthaud

**Comment by antonkomarev** [comment]
@arthaud-proust Hi, that's not because you have a GitHub account. That's because one of the service users follows you or you are following him.

We are taking only minimal public information from GitHub to show your profile, nothing more.

**Reply by arthaud-proust** [reply]
Hi @antonkomarev, 
thanks for the explanation, effectively I have one follower which is on the service
If I remove it, will my profile disappear on your site ?

I understand, but I prefer to not appear on your website
Sorry to ask again, is it possible to remove my profile from the site ?

**Reply by antonkomarev** [reply]
Why is it so important? It's public info, and I'm sure there are many services who are parsing data from the GitHub.

If you unfollow — it wouldn't help. Because we store followage history. But if it is very critical for you, we coild improve service and hide your profile for the unauthenticated users.

**Reply by arthaud-proust** [reply]
Because I choose to display my name and picture on github.com, not on your website. 
And I am not the sole people to make this request. 

Yes please 😶‍🌫️, if you can improve service and hide my profile from everyone and search engine (or better, delete it) I would be grateful

 I thinks it's fair to choose on what website your name appear. You don't want to be on a political website ? Ok right, you wont be (yes it's not about politic here, but you got the principle)

Hope my english is understandable, let me know if my request is hard to fulfill 🙂

**Reply by antonkomarev** [reply]
@arthaud-proust Done. I've implemented a feature to return 403 for people who want to hide their profiles from the guests. Internet crawlers will stop parse your profile and you will disappear from the search engines. But authenticated service users will see you.

Is it enough?

**Reply by arthaud-proust** [reply]
Thanks a lot, you're fast !

If this solution don't block your project then it's good for me
Hope you the best with this project (and it's made with laravel, best choice!)

**Reply by antonkomarev** [reply]
@arthaud-proust Thanks for the feedback!

**Comment by antonkomarev** [comment]
For the future cases: We've implemented a feature to hide accounts from guest users/bots/etc. If anybody want to hide his/her profile just ping me here.

**Reply by tahahasanmasood** [reply]
Could you please tell me why my GitHub information is showing on your page (https://yhype.me/github/). I have not registered at your site. Please remove it.

**Reply by antonkomarev** [reply]
> Could you please tell me why my GitHub information is showing on your page (https://yhype.me/github/). I have not registered at your site. Please remove it.

@tahahasanmasood Done. Your profile is hidden.

**Reply by McMelonTV** [reply]
Hello, please delete my account @antonkomarev

**Reply by victorwynne** [reply]
I want my GitHub account removed from your website. This should be an opt-in service. Terrible business practice.

https://yhype.me/github/@victorwynne

**Reply by antonkomarev** [reply]
@McMelonTV Done

@victorwynne Done

**Reply by karolzlot** [reply]
@antonkomarev please also hide mine, thanks

**Reply by antonkomarev** [reply]
@karolzlot done

**Reply by riyanda-s** [reply]
can you hide me too?

**Reply by antonkomarev** [reply]
@rcsinambela done

**Reply by Errant404** [reply]
can you hide me too?

**Reply by antonkomarev** [reply]
@Errant404 done

**Reply by NanashiTheNameless** [reply]
@antonkomarev Can you please make me hidden to guests/bots too?

**Reply by antonkomarev** [reply]
@NanashiTheNameless done

**Reply by ajm19826** [reply]
can you hide me too?

**Reply by antonkomarev** [reply]
@ajm19826 done

**Comment by antonkomarev** [comment, marked as answer]
Now everyone can control their profile visibility!

Just go the new settings page: https://yhype.me/settings
